src/rex.ctl/README.rst

```diff
@@ -105,14 +105,17 @@ If you wish to name the configuration file differently, or the file is not in
 the current directory, you can use ``--config`` parameter to specify the
 location of the configuration file::
 
-    $ rex deploy --config=/path/to/rex.yaml
+    $ rex deploy --config=myapp.yaml
     ...
 
-    $ rex serve --config=/path/to/rex.yaml
+Alternatively, you can rename the executable to match the name of the
```

Comment:
> you can rename the executable

nit: should we have an example on how to do that in docs? Or are you planning to address that in subsequent PRs to make codebase automatically generate wrappers? (I think it's a good idea as they will show up in shell autocomplete).